src/main/resources/entry-point/rest-mvc/authorization/authorization-jwt.java.mustache

```diff
@@ -0,0 +1,94 @@
+package {{package}}.api.config;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator;
+import org.springframework.security.oauth2.jwt.JwtClaimValidator;
+import org.springframework.security.oauth2.jwt.JwtDecoder;
+import org.springframework.security.oauth2.jwt.JwtValidators;
+import org.springframework.security.oauth2.jwt.NimbusJwtDecoder;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationConverter;
+import org.springframework.security.web.SecurityFilterChain;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Configuration
+@EnableWebSecurity
+@EnableMethodSecurity(prePostEnabled = true)
+public class AuthorizationJwt {
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @Value("${spring.security.oauth2.resourceserver.jwt.issuer-uri}")
+    private String issuerUri;
+    @Value("${spring.security.oauth2.resourceserver.jwt.client-id}")
+    private String clientId;
+    @Value("${jwt.json-exp-roles}")
+    private String jsonExpRoles;
```

Comment:
Bean classes should only have final attributes (constructor injection) to comply with [clean architecture plugin rule 2.7](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Architecture-Rules#rule_27-beans-classes-should-only-have-final-attributes-injection-by-constructor-to-avoid-concurrency-issues)

Reply:
Thank you for the feedback @luisgomez29, I will correct it.